File: AccuPay.Client/src/app/organizations/shared/organization-page-options.ts
import { PageOptions } from 'src/app/core/shared/page-options';

export class OrganizationPageOptions extends PageOptions {
  clientId: number;

  constructor(pageIndex?: number, pageSize?: number, clientId?: number) {
    super(pageIndex, pageSize);
    this.clientId = clientId;
  }

  toObject(): { [param: string]: string } {
    const object = super.toObject();
    if (this.clientId) {
      object.clientId = `${this.clientId}`;
    }

    return object;
  }
}

File: AccuPay.Core/Helpers/OrganizationPageOptions.cs
namespace AccuPay.Core.Helpers
{
    public class OrganizationPageOptions : PageOptions
    {
        public int? ClientId { get; set; }

        public bool HasClientId => ClientId.HasValue;

        public new static OrganizationPageOptions AllData
        {
            get
            {
                var options = new OrganizationPageOptions();
                options.All = true;

                return options;
            }
        }
    }
}